app/src/main/java/com/nervousfish/nervousfish/data_objects/RSAKey.java

```diff
@@ -0,0 +1,62 @@
+package com.nervousfish.nervousfish.data_objects;
+
+/**
+ * RSA variant of {@link IKey}.
+ */
+public final class RSAKey implements IKey {
+
+    private final static String TYPE = "RSA";
```

Comment:
I think this is not that useful. In the POJO it is no problem to have those constants.